src/app/clusters/general-commissioning-server/general-commissioning-server.cpp

```diff
@@ -149,12 +149,12 @@ bool emberAfGeneralCommissioningClusterArmFailSafeCallback(app::CommandHandler *
         CheckSuccess(failSafeContext.ArmFailSafe(accessingFabricIndex, System::Clock::Seconds16(commandData.expiryLengthSeconds)),
                      Failure);
         response.errorCode = CommissioningError::kOk;
-        CheckSuccess(commandObj->AddResponseData(commandPath, response), Failure);
+        CheckSuccess(commandObj->AddResponse(commandPath, response), Failure);
```

Comment:
The CheckSuccess stuff here does not make sense, in the same way that the OTA bits that manually send a status on failure don't make sense.  This one, though, we should fix as part of this PR (by just dropping this CheckSuccess stuff), because otherwise the codesize measurements are fairly meaningless.

Reply:
Removed.